docs/topics/scale.md

```diff
@@ -39,5 +39,5 @@ This command will re-use the `apimodel.json` file inside the output directory as
 |--node-pool|depends|Required if there is more than one node pool. Which node pool should be scaled.|
 |--new-node-count|yes|Desired number of nodes in the node pool.|
 |--master-FQDN|depends|When scaling down a kubernetes cluster this is required. The master FDQN so that the nodes can be cordoned and drained before removal. This should be output as part of the create template or it can be found by looking at the public ip addresses in the resource group.|
-|--auth-method|no|The authentication method used. Default value is 'client_secret'. Other supported values are: 'device' and 'client_certificate'.|
+|--auth-method|no|The authentication method used. Default value is `client_secret`. Other supported values are: `cli`, `client_certificate`, and `device`.|
```

Comment:
is auth-method cli not supported?

Reply:
I think it is https://github.com/Azure/aks-engine/blob/master/cmd/root.go#L222

Reply:
It is supported, but we didn't have it listed here.

Reply:
I read the diff wrong... I'll see my self out ☕️

Reply:
although thinking about it, I think it might be intentional that it's missing because it doesn't work for new users

Reply:
https://github.com/Azure/aks-engine/pull/231

Reply:
or is that device?

Reply:
nevermind, it _is_ supported